src/odemis/driver/test/nenovision_test.py

```
@@ -0,0 +1,458 @@
+
+class TestLiteScope(unittest.TestCase):
+    """Integration tests for the LiteScope actuator driver."""
+
+    @classmethod
+    def setUpClass(cls):
+        if not nenovision:
+            raise unittest.SkipTest("nenovision driver is not available. Check if python3-nidaqmx is installed.")
+        try:
+            cls.scan_stage = LiteScope(**CONFIG_LITESCOPE)
+        except (model.HwError, ValueError) as ex:
+            raise unittest.SkipTest(f"Cannot connect to NI DAQ device: {ex}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)